eng/helix/helix.proj

```diff
@@ -12,7 +12,8 @@
   <Import Project="..\Build.props" />
 
   <ItemGroup>
-    <HelixTargetQueue Include="@(HelixAvailableTargetQueue)" />
+    <HelixTargetQueue Condition="'$(UseHelixOpenQueues)' == 'true'" Include="@(HelixAvailableTargetQueue)" />
+    <HelixTargetQueue Condition="'$(UseHelixOpenQueues)' != 'true'" Include="@(HelixAvailableTargetQueue->'%(Identity)'->Replace('.Open',''))" />
```

Comment:
Why perform the naming substitution twice -- here and in the .target file?

Reply:
They are different things and need to match, one is a set of available queues, and one is the queue to target, the actual id strings contain .Open in our source files

Reply:
Please help me understand why the chosen queue isn't taken from the modified (in internal builds) list of available queues?

Reply:
We have the queue ids with .Open in attributes as well, and those need to be stripped too https://github.com/dotnet/aspnetcore/blob/0517745c08d1bc85009f1ca41a41d77176eeea60/src/Servers/Kestrel/test/FunctionalTests/Http2/HandshakeTests.cs#L83

Reply:
That said, this msbuild stuff is definitely not something I am super familiar with, if you can suggest a way to do this in a different place so we don't have to filter twice, I'm all for trying it

Reply:
I didn't look at how the attribute `Queues` value are merged in. But, one of the following might work:
1. change the `@(HelixAvailableTargetQueue)` item group in place e.g.
    ``` msbuild
    <ItemGroup Condition="'$(_UseHelixOpenQueues)' != 'true'">
      <_Temp Include="@(HelixAvailableTargetQueue)" />
      <HelixAvailableTargetQueue Remove="@(HelixAvailableTargetQueue)" />
      <HelixAvailableTargetQueue Include="@(_Temp->'%(Identity)'->Replace('.Open',''))" />
    </ItemGroup>
    ```
    in your Helix.targets
2. Don't bother w/ the `@(_HelixAvailableTargetQueue)` item group and only update the identities in Helix.proj

Reply:
This didn't work on the internal builds, so I'm reverting back to double filtering.